app/assets/stylesheets/forever_style_guide/modules/_hero_simple.scss

```diff
@@ -0,0 +1,89 @@
+.hero_simple {
+  position: relative;
+  height: 500px;
+  background-size: cover;
+
+  h1, p {
```

Comment:
These should be defined with explicit classes like ex. .hero_simple-heading, hero_simple-subheading or something like that. The text color could also be handled in the markup with color-white, in which case it might make more sense for both of those items to just get a styling modifier, something like hero_simple-shadowed (?)